components/ide/jetbrains/image/download/BUILD.js

```diff
@@ -0,0 +1,48 @@
+const ideConfigs = [
```

Comment:
We can merge it into one image to have one place to define configs?

Reply:
change `components/ide/jetbrains/image/download:intellij` to `components/ide/jetbrains/image/download-intellij` ?

Reply:
sounds good

Reply:
changed